### PR TITLE
reuse getStressModeScriptStep

### PR DESCRIPTION
**Pull request by sejongoh**

### CODE REVIEW

**Comment by sejongoh** [comment]
@mmitche Could your please review the changes? This is simply to reuse genStressModeScriptStep and remove duplicated code.

File: netci.groovy [hunk 1]
```diff
@@ -121,6 +121,9 @@ def static genStressModeScriptStep(def os, def stressModeName, def stressModeVar
         // For these we don't use a script, we use directly
         stepScript += "echo Setting variables for ${stressModeName}\n"
         stressModeVars.each{ k, v -> 
+            if (k == '') {
```
Comment (mmitche) [inline netci.groovy]: When is the key empty?

Comment (sejongoh) [inline netci.groovy]: Corefx baseline scenario doesn't have any environment variables.

```
def static jitStressModeScenarios = ...
...
'corefx_baseline' : ['' : ''], // corefx baseline
```

**Comment by sejongoh** [comment]
@dot

File: netci.groovy (hunk 1 above)
Comment (sejongoh) [inline netci.groovy]: @mmitche Could you take another look?

File: netci.groovy [hunk 2]
```diff
@@ -53,7 +53,7 @@ class Constants {
                'jitstress2_jitstressregs0x10' : ['COMPlus_JitStress' : '2', 'COMPlus_JitStressRegs' : '0x10'],
                'jitstress2_jitstressregs0x80' : ['COMPlus_JitStress' : '2', 'COMPlus_JitStressRegs' : '0x80'],
                'corefx_baseline' : ['' : ''], // corefx baseline
```
Comment (mmitche) [inline netci.groovy]: Ahh in this case, you would do: [:] for an empty map, avoiding the odd k = '' case.

Comment (sejongoh) [inline netci.groovy]: Oh i see. I will update it.

**Comment by sejongoh** [comment]
Waiting for [https://github.com/dotnet/dotnet-ci/issues/209]()

**Comment by sejongoh** [comment]
@dotnet-bot test ci please.

**Comment by sejongoh** [comment]
@mmitche PTAL. I removed checking the empty key.

**Comment by mmitche** [comment]
LGTM

**Pull request merged**